plugins/vpp/ifplugin/vppcalls/vpp1901/ipsec_vppcalls_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpp1901_test
```

Comment:
Please do code cleanup - remove empty lines, commented code, etc.. you can use GoLand tools:
Code -> Reformat Code
Code -> Optimize Imports

Reply:
done